client/src/document/ingredients/browser-compatibility-table/index.tsx

```diff
@@ -34,7 +34,9 @@ const NEW_ISSUE_TEMPLATE = `
 </details>
 `;
 
-function gatherPlatformsAndBrowsers(category): [string[], bcd.BrowserNames[]] {
+function gatherPlatformsAndBrowsers(
+  category: string
```

Comment:
Just a little drive-by edit I snook in when I noticed that parameter didn't have a type. Harmless.